### PR TITLE
Bump version to 2.1.1 to stop warning

### PR DESCRIPTION
**Pull request by olymk2**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -17,7 +17,7 @@
     keywords=["django", "notification" "alerts"],
     packages=find_packages(),
     zip_safe=False,
-    install_requires=["django>=1.11,<2.1"],
+    install_requires=["django>=1.11,<2.1.1"],
```
Comment (benjaoming) [inline setup.py]: This allows version 2.1, which isn't tested, nor in the test matrix. The change would require to edit `tox.init` and `travis.ini` and should instead define a new upper bound as `2.2`.

Comment (olymk2) [reply setup.py]: okay think I have made the changes as described, if they are good and tests pass I will do the same for django-wiki

File: tox.ini [hunk 2]
```diff
@@ -27,7 +27,7 @@ commands =
 deps =
   coverage
   django111: Django>=1.11,<2.0
-  django20: Django>=2.0,<2.1
+  django20: Django>=2.0,<2.2
```
Comment (benjaoming) [inline tox.ini]: Ah yes okay - the lines don't give much guidance... but you would have to ADD django 2.1 testing in the matrix, not overwrite Django 2.0 testing. Let me get that fixed...

**Comment by benjaoming** [comment]
Moved to https://github.com/django-wiki/django-nyt/pull/83

**Pull request closed without merging**